SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -0,0 +1,1436 @@
+%global security_hardening none
+%global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
+%define uname_r %{version}-%{release}
+
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+
+Summary:        Linux Kernel for HCI
+Name:           kernel-hci
+Version:        5.15.63.1
+Release:        1%{?dist}
```

Comment:
Is the version of `kernel-hci` always going to match that of the `kernel`/`kernel-headers` package? If so, please modify the contents of the `version_matching_groups` variable in `toolkit/scripts/check_entangled_specs.py` to reflect that. Feel free to reach out if you need any assistance there.